emsdk_manifest.json

```diff
@@ -145,6 +145,34 @@
     "activated_cfg": "LLVM_ROOT='%installation_dir%/%fastcomp_build_bin_dir%'",
     "cmake_build_type": "Release"
   },
+  {
+    "id": "upstream-clang",
+    "version": "master",
+    "bitness": 32,
+    "install_path": "clang/upstream",
+    "git_branch": "master",
+    "url": "https://github.com/llvm-mirror/llvm.git",
+    "clang_url": "https://github.com/llvm-mirror/clang.git",
+    "lld_url": "https://github.com/llvm-mirror/clang.git",

```

Comment:
Should this be `https://github.com/llvm-mirror/lld.git`?

Reply:
Yes it should... why did I think this worked locally?
Oh wow I had the 64-bit version correct, so it happened to work on my machine.